compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -7576,6 +7576,9 @@ private BType checkMappingField(RecordLiteralNode.RecordField field, BType mappi
                     boolean errored = false;
                     for (BField bField : ((BRecordType) spreadExprType).fields.values()) {
                         BType specFieldType = bField.type;
+                        if (specFieldType.tag ==  TypeTags.NEVER) {
```

Comment:
Why this specific name `specFieldType` here?

Reply:
`fieldName` is already defined in the scope and this is referring to a `type` not a name right?

Reply:
My mistake, It should be `fieldType`

Reply:
`fieldType` is already defined in this scope. https://github.com/ballerina-platform/ballerina-lang/blob/master/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java#L7540